globals.go

```diff
@@ -674,9 +679,9 @@ var (
 	}
 
 	// constant arrays for db update logic
-	AbortableStatuses   = []string{TaskStarted, TaskDispatched}
+	AbortableStatuses   = []string{TaskStarted, TaskDispatched, TaskContainerAllocated}
```

Comment:
Can be rename these task status variables to be prefixed with `Task` (e.g. `TaskAbortableStatuses`) to remain consistent with the other task status global variables?